webapp/src/main/webapp/iaf/frankdoc/views/attribute-description.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <div ng-if="! attr.enum" ng-bind-html="attr.description" />
+  <div ng-if="attr.enum">
+    <div ng-bind-html="attr.description" />
```

Comment:
Het gebruik van de dubbele punt zorgt ervoor dat er geen two-way binding meer is. Met andere woorden, de text wordt eenmalig gevuld en daar blijft het bij.. Nu controleert angular steeds of het 'attr' object gewijzigd is om zo de html code aan te passen. Om er voor te zorgen dat de website snel/responsive blijft is het handig de two-way binding zo veel mogelijk uit te zetten. In dit geval dus voor `attr.description`.

Reply:
Gedaan.